deltachat-rpc-client/tox.ini

```diff
@@ -16,3 +17,15 @@ deps =
     pytest-asyncio
     aiohttp
     aiodns
+
+[testenv:lint]
+skipsdist = True
+skip_install = True
+deps =
+    ruff
+    isort
+    black
+commands =
+    isort --check src/ examples/ tests/
```

Comment:
if I read correctly with ruff, isort is not needed?
(ruff must be configured with `profile="black"`, same as isort)